### ISSUE TITLE
Top panel dropdown lists autohide on mobile (Android, Firefox)

### ISSUE THREAD

**Issue by Dima793** [lukevella/rallly#934]
**Description**
When on a poll webpage in mobile view, tapping any dropdown list/menu button from above the poll tiles (account, Live, Manage) makes them appear for a fraction of a second and then hide back. Some observations:
- First time after loading the webpage, list seems to appear for about half a second. Next times it only appers just for a brief moment. 
- Sometimes the button is also highlighted after the tap, but it's inconsistent. 
- The issue persists when toggling PC view in browser settings for the webpage. 
- Tapping and holding on said dropdown list buttons make the lists appear properly. 
- Dropdown list for participants on one of the poll tiles works properly without having to hold. 

 
**Reproduction steps**
1. Open Firefox browser on Android
2. Open [Rallly](https://rallly.co) website
3. Tap "Get started" button
4. Tap account button (at the top right corner of the webpage)

**Expected behavior**
Dropdown list/menu should be kept expanded until next user action.

**Video**
https://github.com/lukevella/rallly/assets/17815003/cf0b3b02-f793-4de4-9662-69a0f346d66d

**Smartphone:**
 - Device: Asus Zenfone 10
 - OS: Android 13
 - Browser: Firefox 119.1.0

**Comment by lukevella** [comment]
Hi @Dima793, do you get this behavior when using the dropdown menu here as well:

https://ui.shadcn.com/docs/components/dropdown-menu

**Comment by Dima793** [comment]
No, I don't get same behavior there. However, it has other oddities:
My browser is set up to hide URL bar when scrolling down. If I tap Style dropdown list button or Open button when URL bar is hidden, website makes the bar appear again. In that case Style dropdown list button has inconsistent behavior, sometimes same as the above, but sometimes it works properly. Also, when choosing from Style dropdown list, first tap only highlights an option and second tap confirms it.

While on Rallly poll webpage the initially described issue is consistent, happens without hiding URL bar and doesn't make it appear when it's hidden.

**Comment by sharky98** [comment]
I observed the same behavior, but the dropdown stay with a long press. However, this is not the typical expected behavior for mobile users.

Mobile Firefox 122.0.1
Pixel Pro 8
Android 14

I have also made a video, that I am trying to find how to attach here 😅

**Comment by nliautaud** [comment]
Note that the issue is a lot more problematic in the dropdowns where the first entry is an action : the option is instantaneously clicked.

I just deleted two comments by clicking on their `...` settings drop-down.

**Comment by lukevella** [comment]
I've not been able to reproduce this issue myself. Could anyone let me know if this is still happening?

**Comment by Dima793** [comment]
> Could anyone let me know if this is still happening?

Just checked — yes, it does still happen consistently.
Firefox 126.0, Android 14, Asus Zenfone 10.

**Comment by Dima793** [comment]
@lukevella it seems fixed for user dropdown and violent deletion menus, just as mentioned in #1099. But the issue is still present for other menus like the ones for profile actions and Manage (see attached screenshot).
![Screenshot_20240518-074308_Firefox_1](https://github.com/lukevella/rallly/assets/17815003/6ec01d74-639f-4517-8055-a62834a36e66)

**Comment by Dima793** [comment]
UPD: I was wrong, the issue is still present for comment deletion too. Just tried to unfold the menu and the comment was instantly deleted.

**Comment by lukevella** [comment]
@Dima793 thanks for checking. To be honest I'm not sure what is causing this. Seems to be a bug in Firefox Android because it works fine on iOS. If anyone has more information please let me know.

**Comment by lukevella** [comment]
Did a bit more digging and found this issue which seems to be related: https://github.com/radix-ui/primitives/issues/2565

Will try applying the suggested workaround.

**Comment by lukevella** [comment]
The fix should be live now.
